diff.go

```diff
@@ -28,14 +26,26 @@ var _ DiffService = &DiffCmd{}
 type DiffCmd struct {
 	cmd   *exec.Cmd
 	strip int
+	out   []byte
+	done  bool
 }
 
-func NewDiffCmd(cmd *exec.Cmd, strip int) DiffService {
+func NewDiffCmd(cmd *exec.Cmd, strip int) *DiffCmd {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewDiffCmd should have comment or be unexported